src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -3297,6 +3290,14 @@ void ApplicationManagerImpl::UnregisterApplication(
       resume_controller().RemoveApplicationFromSaved(app_to_remove);
     }
 
+    if (IsAppSubscribedForWayPoints(app_id)) {
+      UnsubscribeAppFromWayPoints(app_id);
+      if (!IsAnyAppSubscribedForWayPoints()) {
+        LOG4CXX_ERROR(logger_, "Send UnsubscribeWayPoints");
```

Comment:
Is it an error? I would say Info or debug

Reply:
Fixed in 4fbaf1a5fdec18f20e01d195010f81a62bd7c51a